test/setup/cluster_setup.go

```diff
@@ -400,6 +400,15 @@ func (c *Cluster) createUsersCryptoMaterials() error {
 	return nil
 }
 
+func (c *Cluster) CreateAdditionalUserCryptoMaterials(user string) error {
+	keyPair, err := c.GetX509KeyPair()
+	if err != nil {
+		return err
+	}
+	c.CreateUserCerts(user, keyPair)
+	return nil
+}
```

Comment:
In order to support a user whos cert/key was generated by the other root/intermediate CA, upgrade this method to take also the path to the CAs private key, so you can do this with any CA you want